### PR TITLE
Removed database flags

### PR DESCRIPTION
**Pull request by Lonwabo11**
This change is [Reviewable](https://reviewable.io/reviews/saltastroops/data-archive-database/252)

### CODE REVIEW

File: src/ssda/ssda_populate.py [hunk 1]
```diff
@@ -106,10 +146,6 @@ def validate_options(
             "variable FITS_BASE_DIR)."
         )
 
-    # Can not run insert for a single file
```
Comment (hettlage) [inline src/ssda/ssda_populate.py:109]: This check suggests that ssda populate should not be called with the --file option. So if you remove this check (which you indeed should), you have to get rid of the --file option as well.

**Review by hettlage — CHANGES_REQUESTED** [review]
You have to remove the file ssda/task.py as well. :)

File: src/ssda/ssda_populate.py [hunk 2]
```diff
@@ -121,16 +154,16 @@ def validate_options(
         )
 
     # Either a date range or a FITS file must be specified
-    if not (start and end) and not file:
+    if not (start and end):
         raise click.UsageError(
             "You must either specify a date range (with the --start/--end options) or "
-            "a FITS file (with the --file option)."
+            "a FITS file."
```
Comment (hettlage) [inline src/ssda/ssda_populate.py:160]: "You must specify a start and end date (with the --start/--end options)."

File: src/ssda/ssda_populate.py [hunk 3]
```diff
@@ -121,16 +154,16 @@ def validate_options(
...
         )
 
-    # Date ranges and the --file option are mutually exclusive
-    if (start or end) and file:
+    # Date ranges are mutually exclusive
+    if start or end:
```
Comment (hettlage) [inline src/ssda/ssda_populate.py:164]: The whole if condition has become redundant and should be removed.

File: src/ssda/ssda_populate.py [hunk 4]
```diff
@@ -144,17 +177,14 @@ def validate_options(
         raise click.UsageError("The start date must be earlier than the end date.")
```
Comment (hettlage) [inline src/ssda/ssda_populate.py:177]: The error message is not consistent with the condition checked. Also, there should be an additional check that the start date is earlier than the end date.

**Review by hettlage — CHANGES_REQUESTED** [review]

File: src/ssda/ssda_populate.py (hunk 2 above)
Comment (Lonwabo11) [reply src/ssda/ssda_populate.py:160]: Done

**Review by Lonwabo11 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/saltastroops/data-archive-database/252)* status: 0 of 6 files reviewed, 3 unresolved discussions (waiting on @hettlage)

___
*[src/ssda/ssda_populate.py, line 164 at r2](https://reviewable.io/reviews/saltastroops/data-archive-database/252#-MGOfB_aHbdTPKHEhkcA-r2-164:-MHC1eYiDRLyprV4CrH0:b-896fix) ([raw file](https://github.com/saltastroops/data-archive-database/blob/c3468e1e69367af4e3d7d91cfb618238926380b9/src/ssda/ssda_populate.py#L164)):*
<details><summary><i>Previously, hettlage wrote…</i></summary><blockquote>

The whole if condition has become redundant and should be removed.
</blockquote></details>

Done.
___
*[src/ssda/ssda_populate.py, line 177 at r2](https://reviewable.io/reviews/saltastroops/data-archive-database/252#-MGOfB_aHbdTPKHEhkcA-r2-177:-MHCA17W8WVbQqervNGp:b-896fix) ([raw file](https://github.com/saltastroops/data-archive-database/blob/c3468e1e69367af4e3d7d91cfb618238926380b9/src/ssda/ssda_populate.py#L177)):*
<details><summary><i>Previously, hettlage wrote…</i></summary><blockquote>

The error message is not consistent with the condition checked. Also, there should be an additional check that the start date is earlier than the end date.
</blockquote></details>

Done.

**Review by hettlage — APPROVED** [review]
:lgtm:

> Reviewed 4 of 5 files at r1, 1 of 2 files at r3, 1 of 1 files at r5.
> *[Reviewable](https://reviewable.io/reviews/saltastroops/data-archive-database/252#-:-MHuBMNWBCUHJb6I0YVG:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @hettlage)

**Review by hettlage — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/saltastroops/data-archive-database/252#-:-MHuCLsO8Cqc4NEw7kCp:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**